src/v0.rs

```diff
@@ -1261,17 +1261,17 @@ mod tests {
     macro_rules! t_const {
         ($mangled:expr, $value:expr) => {
             t_nohash!(
-                concat!("_RIC0K", $mangled, "E"),
-                concat!("::<", $value, ">")
+                concat!("_RIC8my_crateK", $mangled, "E"),
+                concat!("my_crate::<", $value, ">")
             )
         };
     }
     macro_rules! t_const_suffixed {
         ($mangled:expr, $value:expr, $value_ty_suffix:expr) => {{
             t_const!($mangled, $value);
             t!(
-                concat!("_RIC0K", $mangled, "E"),
-                concat!("[0]::<", $value, $value_ty_suffix, ">")
+                concat!("_RIC8my_crateK", $mangled, "E"),
+                concat!("my_crate::<", $value, $value_ty_suffix, ">")
```

Comment:
I think it's more in the original spirit to keep the old mangling for these tests and remove the `[0]`, because now the output is even closer to what I originally intended (i.e. `C0` is used as a dummy empty path to "parameterize").

`t_nohash_type` does something similar but instead of generics it uses one of the `impl` forms (`M`).

Reply:
Sure, I'll fix that.

Reply:
Should be fixed now.